apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -13,10 +13,11 @@ const e2eDelegationAccounts = [
     delegate: new Delegate(Account.SOL_1, "0.001", "Ledger by Figment"),
     xrayTicket: "B2CQA-2742",
   },
-  {
-    delegate: new Delegate(Account.NEAR_1, "0.01", "ledgerbyfigment.poolv1.near"),
-    xrayTicket: "B2CQA-2741",
-  },
+  //ToDo: enable when speculos interactions are refactored by @victor
```

Comment:
maybe let's link a ticket ?

Reply:
@VicAlbr can you please provide one?

Reply:
Yep, I'm doing the refacto on this ticket: QAA-306

Reply:
thanks, ticket added to the comment